test/int64_test.dart

```diff
@@ -457,23 +460,20 @@ void main() {
       expect(n1 & n2, Int64(1168));
       expect(n3 & n2, Int64(8708));
       expect(n4 & n5, Int64(0x1034) << 32);
-      expect(() => n1 & null, throwsArgumentError);
```

Comment:
Should these be moved to a separate test file which is opted out of null safety?

Reply:
We could, it felt like a bit overboard though. These are pretty defensive anyways and the apis do explicitly check for null.

Reply:
As long as we are publishing as a breaking version change I'm not concerned with going out of the way to preserve the exact behavior in opt-out mode.